src/test/java/steps/MenuBtnsSteps.java

```diff
@@ -1,29 +1,48 @@
 package steps;
 import io.cucumber.java.en.*;
+
+import java.util.List;
+
 import org.junit.Assert;
+import org.openqa.selenium.WebElement;
+
 import pages.MenuBtnsPage;
 
 public class MenuBtnsSteps {
 
     MenuBtnsPage menuBtnsPage = new MenuBtnsPage(); 
-        @Given("I navigate to {string}")
+        @Given("I navigating on {string}")
```

Comment:
Te voy a pasar un documento con las expresiones regulares básicas de matcheo para usar en estos casos. Por ejempli acá: Si pones: @Given("I navigate (.*) {string}") esta expresion te matchea tanto con "I navigate to" como con "I navigate on". Entonces usas la misma función para ambos casos dependiendo a como lo necesites en el gherkin.